library/core/tests/slice.rs

```diff
@@ -1392,6 +1419,13 @@ mod slice_index {
             bad: data[..= usize::MAX];
             message: "maximum usize";
         }
+
+        in mod boundpair_overflow {
+            data: [0; 1];
+
+            bad: data[(Bound::Unbounded, Bound::Included(usize::MAX))];
```

Comment:
We could also include `(Bound::Excluded(usize::MAX), Bound::Unbounded)` here.

Reply:
Done